UI/ControlMappingScreen.cpp

```diff
@@ -337,6 +337,9 @@ bool KeyMappingNewKeyDialog::key(const KeyInput &key) {
 		if (key.keyCode == NKCODE_EXT_MOUSEBUTTON_1) {
 			return true;
 		}
+		// Only map analog values to this mapping.
+		if (pspBtn_ == VIRTKEY_SPEED_ANALOG)
```

Comment:
Should we provide a message when user try to bind a key?

Reply:
Well, it just doesn't react.

-[Unknown]

Reply:
Mhhhh, it should react to at least "Back" button. If a controller have no analog axis this will brick the emulation basically.

Reply:
Ah, I thought it did allow a way to cancel without touch already.  Well, I let cancel go through.  Thanks for pointing out.

-[Unknown]